caffe2/python/onnx/tests/conversion_test.py

```diff
@@ -241,33 +241,6 @@ def _make_fake_loop_op(self, body_nodes, input_types, output_types):
         ]
         return retval_nodes
 
-    def test_onnx_to_caffe2_loop(self):
```

Comment:
@jacky82226 - I am curious why this has to be removed. Does this test point have any dependency on ONNX optimizer?

Reply:
It is for an experiment. 
Actually there are a few tests under caffe2/python/onnx/tests written after introducing optimizer for onnx-caffe2 conversion. If I simply remove the optimization part in onnx-caffe2, there are a few tests broken. It seems that the onnx.optimizer modifies the graph and the testing codes test that part as well.